### ISSUE TITLE
Bad links on 2017-06-06/personalization-semantics-10-first-public-working-draft/

### ISSUE THREAD

**Issue by SteveALee** [w3c/wai-news#6]
```
<a href="https://www.w3.org/personalization-semantics-1.0/">Personalization Semantics 1.0</a>
```
is 404

```
<a href="https://www.w3.org/WAI/GL/">Accessible Rich Internet Applications Working Group</a>
```
is to AGWG not ARIA WG

**Comment by yatil** [comment]
> <a href="https://www.w3.org/WAI/GL/">Accessible Rich Internet Applications Working Group</a>
is to AGWG not ARIA WG

I fixed that one in 5abc16dec2db7de275e273d9799d6282478a28dc

**Comment by slhenry** [comment]
yup just missing TR https://www.w3.org/TR/personalization-semantics-1.0/
